app/preload/accountReport.ts

```diff
@@ -24,18 +25,24 @@ import { getIdentityVersion } from './database/identityDao';
 import { convertIncomingTransaction } from '~/utils/TransactionConverters';
 import httpMethods from './http';
 import decryptAmountsDao from './database/decryptedAmountsDao';
-import decryptTransactions, {
-    isSuccessfulEncryptedTransaction,
-} from '~/utils/decryptHelpers';
 import { hasEncryptedBalance } from '~/utils/accountHelpers';
+import isSuccessfulEncryptedTransaction from '~/utils/decryptHelpers';
 
 async function enrichWithDecryptedAmounts(
     credentialNumber: number,
     prfKeySeed: string,
     identityVersion: IdentityVersion,
     address: string,
     global: Global,
-    transactions: TransferTransaction[]
+    transactions: TransferTransaction[],
+    decryptTransactions: (
+        encryptedTransfers: TransferTransaction[],
+        accountAddress: string,
+        prfKey: string,
+        identityVersion: IdentityVersion,
+        credentialNumber: number,
+        global: Global
```

Comment:
Yes, this is exactly why. If we use a web worker from the preload script it compiles a specific preload.worker. Since we compile the preload script with access to node it compiles the worker in a way that requires node, which is disabled for web workers for security reasons.